NineChronicles.Headless.Executable/Commands/ActionCommand.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections;
+using System.IO;
+using Bencodex;
+using Bencodex.Types;
+using Cocona;
+using Libplanet;
+using Nekoyume.Model;
+using NineChronicles.Headless.Executable.IO;
+
+namespace NineChronicles.Headless.Executable.Commands
+{
+    public class ActionCommand : CoconaLiteConsoleAppBase
+    {
+        private static readonly Codec Codec = new Codec();
+        private readonly IConsole _console;
+
+        public ActionCommand(IConsole console)
+        {
+            _console = console;
+        }
+
+        [Command(Description = "Create ActivateAccount action.")]
+        public int ActivateAccount(
+            [Argument("INVITATION-CODE", Description = "A hex-encoded invitation code.")] string invitationCode,
+            [Argument("NONCE", Description = "A hex-encoded nonce for activation.")] string nonceEncoded,
+            [Argument("path", Description = "A file path of hex encoded action.")] string filePath
+        )
+        {
+            try
+            {
+                ActivationKey activationKey = ActivationKey.Decode(invitationCode);
```

Comment:
Could you write unittest for the case if the user passed invalid `invitationCode`?